x-pack/plugins/security_solution/public/resolver/store/data/selectors.ts

```diff
@@ -109,8 +109,16 @@ export const terminatedProcesses = createSelector(resolverTreeResponse, function
  * Process events that will be graphed.
  */
 export const graphableProcesses = createSelector(resolverTreeResponse, function (tree?) {
+  // Keep track of the last process event (in array order) for each entity ID
```

Comment:
Sorry if these are silly questions:

1. Is this expecting the lifecycle events array to be sorted by time?
    1. If so does that sorting need to be done by the backend or does the front end ensure that the array is sorted?
    2. If it is not expecting the array to be sorted, what happens if the `end` lifecycle events appears first in the array for a specific `entityID` and then the `start` event comes last for that same `entityID`?

Reply:
So this only accepts 'graphable' ones: https://github.com/elastic/kibana/blob/05b07893121d33aed3599b05effa7e6c634e211b/x-pack/plugins/security_solution/public/resolver/models/process_event.ts#L15

and it would take the last one. So if there was like 
`info, info, end ` it would get the second one